moveit_core/robot_trajectory/src/robot_trajectory.cpp

```diff
@@ -630,6 +630,32 @@ void RobotTrajectory::print(std::ostream& out, std::vector<int> variable_indexes
   out.flush();
 }
 
+bool RobotTrajectory::isContinuous(const double threshold) const
+{
+  bool is_continuous = true;
```

Comment:
Nit: Can we return true/false instead of assigning value to this variable and returning at the end?

Reply:
It makes sense to print all discontinuities instead of early termination after detecting the first discontinuity.